### PR TITLE
Generate 18 character password for admin user

### PR DESCRIPTION
**Pull request by tillkruss**
This matches the default password length for new WP installs.

See also: https://github.com/wp-cli/entity-command/pull/7

### CODE REVIEW

File: php/commands/core.php [hunk 1]
```diff
@@ -535,7 +535,7 @@ function wp_new_blog_notification() {
 			'title' => '',
 			'admin_user' => '',
 			'admin_email' => '',
-			'admin_password' => ''
+			'admin_password' => wp_generate_password(18)
```
Comment (danielbachhuber) [inline php/commands/core.php]: This approach will cause the admin password to never display. See later on:

```
if ( empty( $admin_password ) ) {
 			WP_CLI::log( "Admin password: {$result['password']}" );
}
```

The expected behavior is that `Admin password:` displays if no password is provided.

You'll need to modify your logic to meet this expected behavior. If this doesn't fail tests, you should also add a test for this.

Comment (tillkruss) [inline php/commands/core.php]: Thanks will fix that. Just that the 15min CI... holy glob!

**Comment by danielbachhuber** [comment]
Thanks @tillkruss 👍

**Comment by tillkruss** [comment]
Thanks @danielbachhuber!

**Pull request merged**